PRS6001-7000/PRS6427Macarius.xml

```diff
@@ -65,11 +66,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                <death when="0391">391</death>
                <floruit>Alongside <persName ref="PRS1962Anthony">Anthony</persName> and <persName ref="PRS7696Pachomiu">Pachomius</persName>, he ranks among the most illustrious Egyptian desert fathers.
                In Ethiopian tradition he is featured in various translated monastic collections such as <ref type="work" corresp="LIT2611Zenaab"/> or <ref type="work" corresp="LIT4838CollectioM"/> and in the so-called
-               monastic genealogies. He is also venerated as saint and mentioned in the <ref type="work" corresp="LIT2375Synaxa">Synaxary</ref>
+               monastic genealogies. He is also venerated as saint and mentioned in the <ref type="work" corresp="LIT2375Synaxa">Synaxary</ref>.
+                  Not to be confused with his contemporary <persName ref="PRS14587Macarius">Macarius of Alexandria</persName>.
                </floruit>
 <note type="commemoration"><date type="diesNatalis" ref="ethiocal:Magabit27">27 Maggābit</date></note>
```

Comment:
I think there was a guideline that diesNatalis is used only when it reflects the reality, Macarius is venerated on the date of his death.